app/controllers/people_controller.rb

```diff
@@ -16,7 +16,10 @@ def show
         response.headers['Content-Length'] = @person.v_card.size.to_s
       }
       format.pdf {
-          pdf = Services::Token.generate_token_pdf(@person, settings.token_media)
+          Rails.logger.info "settings #{settings.token_media}"
+          
+          pdf = Services::Token.generate_token_pdf(@person, settings.token_media,
+                                        ServiceProvider.find_by_special_role(:registration).token_layout)
```

Comment:
Agreed. The print wizard stuff could be used to generate specific paper forms for the service provider. Alternatively we could try using a report generator package. I am reasonable comfortable with the ability to layout at least simple single page forms using the current structure. These user stories need to be developed further.